runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -0,0 +1,158 @@
+local M = {}
+local util = require "vim.lsp.util"
+local ns = vim.api.nvim_create_namespace("lsp-semantic-tokens")
+
+local semantic_tokens = {}
+
+local function modifiers_from_number(x, modifiers_table)
+  local function get_bit(n, k)
+    -- Based on/from https://stackoverflow.com/a/26230537
+    -- (n >> k) & 1
+    return _G.bit.band(_G.bit.rshift(n, k), 1)
```

Comment:
TODO: is this portable outside of LuaJIT

Reply:
No...

Reply:
But see https://github.com/neovim/neovim/issues/11352

Reply:
We could upstream https://github.com/nvim-lua/plenary.nvim/blob/bfe08f26bdc246a13f66e37d44334da83c27f134/lua/plenary/bit.lua for this.

Reply:
I provided a fall-back implementation until `bit` is available for all Lua implementation in nvim